spec/classroom_spec.rb

```diff
@@ -0,0 +1,11 @@
+require './classroom'
+
+describe Classroom do
+  context 'create a classroom' do
+    classroom = Classroom.new('ruby language')
+
+    it 'show classroom' do
+      expect(classroom.label).to eq 'ruby language'
+    end
+  end
+end
```

Comment:
- [ ] You need to test every method on every class, here kindly check the add_student method
![image](https://user-images.githubusercontent.com/62705947/234839588-c490bc84-48ce-4986-a7a5-4a13f2808b0f.png)

Reply:
done writing test